.travis.yml

```diff
@@ -2,3 +2,7 @@ sudo: false
 language: node_js
 node_js:
   - "node"
+install:
+  - npm install
+script:
+  - 'npm run test:eslint'
```

Comment:
This test doesn't exist yet in `package.json`. It need something like [this](https://github.com/ismay/ismay.github.io/blob/development/package.json#L13)